source/_components/mqtt.markdown

```diff
@@ -119,6 +119,10 @@ mqtt:
 There is an issue with the Mosquitto package included in Ubuntu 14.04 LTS. Specify `protocol: 3.1` in your MQTT configuration to work around this issue.
 </p>
 
+<p class='note'>
+If you are running a mosquitto instance on a different server with proper SSL encryption using a service like letsencrypt you may have to set the certificate to the operating systems own `.crt` certificates file. In the instance of ubuntu this would be `certificate: /etc/ssl/certs/ca-certificates.crt`
```

Comment:
Thanks for the feedback :slightly_smiling_face: the only thing related I can find is the this [superuser](http://superuser.com/questions/437330/how-do-you-add-a-certificate-authority-ca-to-ubuntu) link, I don't really see anyone else using a dedicated external server for their MQTT broker and a dedicated internal server at home that isn't a raspberry pi, so I am struggling to find anything :disappointed: